modules/andes-core/broker/src/main/java/org/wso2/andes/server/information/management/SubscriptionManagementInformationMBean.java

```diff
@@ -55,6 +55,11 @@
     private static final String NOT_AVAILABLE = "N/A";
 
     /**
+     * Used to non local subscription protocol queue name
+     */
+    private static final String EMPTY_NAME = "";
```

Comment:
Can't we use the same NOT_AVAILABLE string defined above for this. It will make the code look clean :)